tools/integration_tests/operations/operations_test.go

```diff
@@ -88,7 +88,7 @@ const Content = "line 1\nline 2\n"
 const onlyDirMounted = "OnlyDirMountOperations"
 
 func createMountConfigsAndEquivalentFlags() (flags [][]string) {
-	cacheDirPath := path.Join(os.Getenv("HOME"), "cache-dri")
+	cacheDirPath := path.Join(os.Getenv("HOME"), "cache-dir")
```

Comment:
Read cache package is also using same directory for caching. And both operations and read cache are running in parallel, it might cause inconsistency and flakiness.

Reply:
Updated the name